lib/api/controllers/documentController.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
-var
+let
   BadRequestError = require('kuzzle-common-objects').errors.BadRequestError,
+  SizeLimitError = require('kuzzle-common-objects').errors.SizeLimitError,
   assertHasBody = require('./util/requestAssertions').assertHasBody,
   assertHasId = require('./util/requestAssertions').assertHasId,
   assertBodyHasAttribute = require('./util/requestAssertions').assertBodyHasAttribute,
   assertHasIndexAndCollection = require('./util/requestAssertions').assertHasIndexAndCollection,
   assertBodyAttributeType = require('./util/requestAssertions').assertBodyAttributeType,
   Request = require('kuzzle-common-objects').Request,
+  Promise = require('bluebird'),
```

Comment:
would be nice to think about not override default javascript object (`Promise` is a build-in object)